src/connectors/bilibili.ts

```diff
@@ -0,0 +1,504 @@
+export {};
+
+/*
+	Since BiliBili is a video site, accurately scrobbling song information from the title can be challenging. 
+	However, people often include the artist's name and track title in the video tags. 
+
+	Therefore, this connector uses tags to identify artist and track information and determines whether we should scrobble this video.
+	The video title is then used to determine which tag is most likely to contain the artist or track name.
+*/
+
+/**
+ * Quick link to debug and test the connector: https://www.bilibili.com/video/BV1LW4y1g7kN
+ *
+ * the scrobble info will shown beside the video info, under the video title.
+ *
+ * special behaviors:
+ * if the video has bgm tag(a tag with music symbol in front of it), it'll be recognized as track name
+ * if the video has no track or artist related tag, this connector will try grepping it from video title
+ */
+
+/**
+ * selectors used by this connector
+ */
+const videoTitleSelector = '.video-title';
+const videoSelector = 'video';
+const tagPanelSelector = '.tag-panel';
+const tagLinkSelector = '.tag-link';
+const uploaderSelector = '.up-name';
+const bgmTagSelector = '.bgm-tag .tag-link .tag-txt';
+
+/**
+ * selectors used by Connector core
+ */
+const PlayerSelector = '.bpx-player-container';
+const currentTimeSelector = '.bpx-player-ctrl-time-current';
+const durationSelector = 'bpx-player-ctrl-time-duration';
```

Comment:
```suggestion
const durationSelector = '.bpx-player-ctrl-time-duration';
```

Reply:
Fixed. Sorry for the mistake 😥 thank you for pointing it out!